backend/src/validators/picker_options.js

```diff
@@ -0,0 +1,21 @@
+const getOneValidator = {
+  // Request parameters.
+  params: {
+    id: { type: "string" },
+  },
+  // Response validation.
+  response: {
+    200: {
+      type: "object",
+      properties: {
+        data: {
+          id: { type: "integer" },
```

Comment:
Maybe we should add a bit more to this validator since picker options should be relatively uncomplicated and unlikely to change their structure, unless I'm missing something. What is the actual data being returned here? Wouldn't it just be an array of strings? Can we put in validation for that?

I guess I just want to be cautious about adding a validator that only checks for an ID(which all DB entries should have, regardless of the other data) and not adding any more validation. If this is just a basic validator for now as an MVP, when will more specific validation be added?

Reply:
Thanks for the comment! This is only for MVP that's why it's so basic.